From 4449f95bbbea6ecce35118ac9e408580e182fed1 Mon Sep 17 00:00:00 2001
From: sk <sk@typedivision.de>
Date: Sat, 10 Nov 2018 22:48:39 +0000
Subject: [PATCH] no license deployment for ct itself

---
 scripts/functions | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/scripts/functions b/scripts/functions
index 491c7595..b75a52dd 100644
--- a/scripts/functions
+++ b/scripts/functions
@@ -2421,12 +2421,5 @@ CT_InstallCopyingInformation()
         done
     done
 
-    # Also add crosstool's information
-    for licfile in ${CT_TOP_DIR}/{COPYING*,LICENSE*,licenses.d}; do
-        dstdir="${CT_PREFIX_DIR}/share/licenses/crosstool-ng"
-        mkdir -p "${dstdir}"
-        CT_DoExecLog ALL cp -av "${licfile}" "${dstdir}/"
-    done
-
     shopt -u nullglob
 }
-- 
2.19.1

